example/src/components/subsection.js

```diff
@@ -2,22 +2,28 @@
 import React, { PropTypes } from 'react';
 import MD  from './md';
 
-
+/*
+ * Represents some site content
+ */
 export default function Subsection(props) {
     const {
         title='',
-        markdown,
         link,
         subSections=[],
-        code: Code
+        content
     } = props;
 
     return <div id={link} className="subsection">
         {title && title.length && <h3>{title}</h3>}
 
-        {markdown && <MD text={markdown} />}
-
-        {Code && <Code />}
+        {content && content.map(Content => {
```

Comment:
You will need to add a `key` prop to `MD` or `Content` to stop react from complaining. I would pull in the second map param (`index`, or `i`) and use that as the key.